sparklingpandas/utils.py

```diff
@@ -7,6 +7,21 @@
 from glob import glob
 
 
+def setup_pyspark():
```

Comment:
I don't like that every py class needs this method at the start. We should be able to modify the path more globably. Does it really need to happen for every class? Could we instead make this module-level initialization, meaning in __init__.py?